lib/templates.ts

```diff
@@ -129,6 +138,17 @@ export async function createTemplate(command: CreateTemplateCommand): Promise<Fo
     throw e;
   });
 
+  const validationResult = validateTemplate(command.formConfig);
+
+  if (!validationResult.valid) {
+    logMessage.warn(
+      `[templates][createTemplate] Form config is invalid. ${JSON.stringify(
```

Comment:
The goal here is just to monitor how often it happens. We can remove it in the future.